Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -110,6 +110,39 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(try self.coll.insertOne(self.doc1)).toNot(throwError())
     }
 
+    func testDropWithWriteConcerns() throws {
```

Comment:
I think this test can be combined with `testDrop`, similar to how you have a single test for the database drop.